### ISSUE TITLE
Can you please split the SDK?

### ISSUE THREAD

**Issue by orgads** [yandex-cloud/nodejs-sdk#152]
I only use ai, which is 1M, yet the installed size is 66M.

Is it possible to have separate packages, similar to aws-sdk v3?

**Comment by DavyJohnes** [comment]
Good idea, but not easy to implement. As a alternative solution I would suggest to use any bundler (weback, for example) which is able to "shake dependency tree" and add only actual used code. It is useful not only for yc nodejs-sdk, but for all dependencies as well

**Comment by evilandfox** [comment]
I also found that after installing this sdk tests execution takes 5x longer time

**Comment by DavyJohnes** [comment]
> I also found that after installing this sdk tests execution takes 5x longer time

Relation between sdk size and tests is not obvious. Need more details on this issue. What tests? How it runs? How cloud-sdk is involved in your tests?

**Comment by orgads** [comment]
Why was this closed?
